e2e/stack_test.go

```diff
@@ -441,6 +447,97 @@ func rollupFlow(t *testing.T, stack *e2e.StackConfig) {
 	t.Log("Monomer can initiate withdrawals on L2 and can generate proofs for verifying the withdrawal on L1")
 }
 
+func erc20RollupFlow(t *testing.T, stack *e2e.StackConfig) {
+	// TODO: debug race condition with ethRollupFlow/erc20RollupFlow
+	time.Sleep(2 * time.Second)
```

Comment:
question to reviewers - I haven't had any luck in fixing the race condition without this `time.Sleep`. I believe that the race condition occurrs because the `ethRollupFlow/erc20RollupFlow` L1 tx nonces get out of order. To avoid focusing too much on this one particular issue, wdyt about just leaving in this `time.Sleep` for a staggered start to avoid the race condition?

Reply:
I think we should go ahead and solve this. Having functional e2e tests will be the basis of our future work to make sure we don't break anything.

The nonce issue makes sense. To confirm, we can try using a different user than stack.User. Let's just put the whole Secrets struct on the StackConfig and use Secrets.Alice and Secrets.Bob for different tests.

Reply:
Good point, I feel like that would be a quick fix that would resolve the race condition. I'll look into this